vault/logical_system.go

```diff
@@ -553,8 +553,16 @@ func (b *SystemBackend) handlePluginCatalogUpdate(ctx context.Context, _ *logica
 
 	command := d.Get("command").(string)
 	ociImage := d.Get("oci_image").(string)
-	if command == "" && ociImage == "" {
-		return logical.ErrorResponse("must provide at least one of command or oci_image"), nil
+	var resp logical.Response
+
+	if sha256 == "" {
```

Comment:
nit: would it be possible to collapse this into a single `if {} else if {}` statement?